force-app/main/default/flows/Process_Charity_Amount.flow-meta.xml

```diff
@@ -46,6 +46,13 @@
                 <booleanValue>true</booleanValue>
             </value>
         </filters>
+        <filters>
+            <field>Id</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>$Record.ReferenceEntityId</elementReference>
```

Comment:
Why are you changing this flow in this PR? if you must do this change you should create a new PR for the correspondent  exercise.